### PR TITLE
add predy v5

### PR DESCRIPTION
**Pull request by wshino**

### CODE REVIEW

File: fees/predy-finance/index.ts [hunk 1]
```diff
@@ -309,10 +428,16 @@ const adapter: BreakdownAdapter = {
     //     start: async () => 1671092333,
     //   },
     // },
-    v320: {
+    // v320: {
```
Comment (dtmkeng) [inline fees/predy-finance/index.ts]: could you set like this is this version is not support 
https://github.com/DefiLlama/dimension-adapters/blob/master/protocols/pancakeswap/index.ts#L120

**Review by dtmkeng — CHANGES_REQUESTED** [review]

**Comment by llamatester** [comment]
The predy-finance adapter exports: 
        
 
 _______________________________________
    Fees for 1/8/2023
    _______________________________________
    
    Version -> V3
    ---------
    ARBITRUM 👇
    Backfill start time not defined
    NO METHODOLOGY SPECIFIED
    Timestamp: 0
    
    
    Version -> V320
    ---------
    ARBITRUM 👇
    Backfill start time not defined
    NO METHODOLOGY SPECIFIED
    Timestamp: 0
    
    
    Version -> V5
    ---------
    ARBITRUM 👇
    Backfill start time: 4/7/2023
    NO METHODOLOGY SPECIFIED
    Timestamp: 1690934399
    Daily fees: 8.658682020558239009
    Daily revenue: undefined
    Daily supply side revenue: undefined
    
    
    Done in 7.73s.

File: fees/predy-finance/index.ts (hunk 1 above)
Comment (wshino) [reply fees/predy-finance/index.ts]: @dtmkeng Thank you for reviewing, I've disabled v3 and v3.2.

**Comment by waynebruce0x** [comment]
@wshino the graph endpoint has been removed - will this be maintained in the future? we can no longer track data. thanks

**Pull request merged**